features/steps/steps.py

```diff
@@ -283,4 +294,19 @@ def step_impl(context, representation_id):
                     errors.append(representation_shape_error(inst, representation_id))
     
     handle_errors(context, errors)
+
+@then('Each {entity} may be nested by only the following entities: {other_entities}')
+def step_impl(context, entity, other_entities):
+
+    allowed_entity_types = other_entities.split(', ')
```

Comment:
I would prefer:

    set(map(str.strip(), other_entities.split(','))

- Remove white space after splitting to be a bit more whitespace agnostic.
- Convert to set immediately.

Reply:
https://github.com/buildingSMART/ifc-gherkin-rules/commit/433fe4ca8d9318574873a26542a6f321856aacd1